manim/mobject/svg/text_mobject.py

```diff
@@ -399,6 +399,7 @@ def __init__(
         stroke_width: int = 0,
         color: str = WHITE,
         size: int = 1,
+        font_size=48,
```

Comment:
Users would be confused by having both `size` and `font_size` parameters.

Reply:
> I would like it to be called scale_factor.

Even though this parameter is effectively just scaling, I think the idea of a `font_size` is something that most people can easily grasp and work with, especially when dealing with text. Renaming the parameter to `scale_factor` would be less intuitive. In my opinion, how it really works under the hood isn't the biggest concern, because if the user can see that it works like a `font_size` and it meets their expectations, then it's acceptable.

> Also, Pango can handle Font Size (the size parameter in Text)

We can have `Text` use the appropriate method of handling Font Size when a fix is implemented.

> Users would be confused by having both size and font_size parameters.

Since `size` isn't working at the moment, I don't think there are many people that are using it to adjust their text. We can probably add some documentation to explain the differences between the two.

Reply:
> Even though this parameter is effectively just scaling, I think the idea of a font_size is something that most people can easily grasp and work with, especially when dealing with text. Renaming the parameter to scale_factor would be less intuitive. In my opinion, how it really works under the hood isn't the biggest concern, because if the user can see that it works like a font_size and it meets their expectations, then it's acceptable.

Changing font size is never the same as scaling. There are ways to change font size Tex also but that isn't implemented https://texblog.org/2012/08/29/changing-the-font-size-in-latex/ and for `Text`, it is already there as `size` parameter(maybe you can rename them to `font_size` if you want to). 

> We can have Text use the appropriate method of handling Font Size when a fix is implemented.

It isn't broken entirely. For example, this works
```py
def construct(self):
  self.texts = [Text(f'hello{i}',size=i) for i in range(1,30)]
  for i in self.texts:
    self.add(i.scale(0.4))
    self.wait()
    self.clear()
```

https://user-images.githubusercontent.com/49693820/122533767-8a7e9280-d03f-11eb-8e8c-21c111ce0465.mp4

And I don't want to confuse users by introducing another parameter.

If the end goal is to increase the font size there are ways to do that currently rather than scaling.

Reply:
It's reasonable to not want to introduce a new parameter. 

I noticed [this commit](https://github.com/3b1b/manim/commit/6eb7edc664d48498fa53aefabf838c2aa80782fa) by you from the manimgl branch regarding text scaling. Would you instead prefer deprecating `size` and using the built in scaling as opposed to scaling? 

I can also look into the Tex font size, but from what I've seen it's generally preferred to not use strict numbers for font_size and instead something like `\huge`, `\small` etc.. It is possible to use `\fontsize` as described [here](https://www.sascha-frank.com/latex-font-size.html) but it can also cause errors when a font is not supported. Also, scaling in math mode seems to be less straightforward and seems like it entails scaling [from stackexchange](https://tex.stackexchange.com/questions/60453/reducing-font-size-in-equation). So, I think having `font_size` simply scale text is fine for `MathTex`/`Tex`

Reply:
> Would you instead prefer deprecating size and using the built in scaling as opposed to scaling?

Fine with me.

> I can also look into the Tex font size, but from what I've seen it's generally preferred to not use strict numbers for font_size and instead something like `\huge`, `\small` etc.. It is possible to use `\fontsize` as described [here](https://www.sascha-frank.com/latex-font-size.html) but it can also cause errors when a font is not supported. Also, scaling in math mode seems to be less straightforward and seems like it entails scaling: [from stackexchange](https://tex.stackexchange.com/questions/60453/reducing-font-size-in-equation). So, I think having `font_size` simply scale text is fine for `MathTex`/`Tex`

Not sure about this. what do you this about this @ManimCommunity/latex?

Reply:
Scaling is fine for now. It usually isn't a big problem to use `\fontsize{x}{y}\selectfont`, but this PR shouldn't be delayed over it. We can investigate / think about it later.